src/Compilers/CSharp/Portable/Binder/Binder_Await.cs

```diff
@@ -40,7 +47,7 @@ private BoundExpression BindAwait(BoundExpression expression, SyntaxNode node, D
                 hasErrors ? CreateErrorType() :
                 Compilation.DynamicType;
 
-            return new BoundAwaitExpression(node, expression, getAwaiter, isCompleted, getResult, awaitExpressionType, hasErrors);
+            return (new AwaitableInfo(getAwaiter, isCompleted, getResult, awaitExpressionType), hasErrors);
```

Comment:
>awaitExpressionType [](start = 74, length = 19)

I think we can return ```awaitExpressionType``` separately from AwaitableInfo, it looks like only ```BindAwait``` needs it.